storage/postgres/migrations/009_notifications.up.sql

```diff
@@ -0,0 +1,51 @@
+BEGIN;
+
+CREATE TYPE notification_type AS ENUM ('CREATED', 'DELETED', 'MODIFIED');
+
+CREATE TABLE notifications
+(
+  id            varchar(100) PRIMARY KEY,
+  resource      varchar(100) NOT NULL,
+  type          notification_type NOT NULL,
+  platform_id   varchar(100) REFERENCES platforms (id) ON DELETE CASCADE, -- value is platform_id from platforms table or null
+  revision      bigserial    NOT NULL,
+  created_at    timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at    timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  new           json, -- json with the new value of the object and additional properties
```

Comment:
wouldnt it be better to have object json column here and we can put new, old and label changes in it ? that way if we need to change the notification format in the future we won't have to change the db table